### PR TITLE
Add a ‘link’ icon next to borrowed series in the module navigation

### PR DESCRIPTION
**Pull request by Coenego**

### CODE REVIEW

**Comment by bp323** [comment]
Putting on hold as there's a backend dependency

**Comment by Coenego** [comment]
Depending on https://github.com/fronteerio/grasshopper/pull/63

**Comment by Coenego** [comment]
Assigning to @bp323 for review

**Comment by bp323** [comment]
Looks good, will wait for the backend PR to be merged.

File: apps/timetable/admin/ui/css/admin.css [hunk 1]
```diff
@@ -394,6 +394,12 @@
     display: block;
 }
 
+#gh-batch-edit-container > i.fa-link{
+    font-size: 18px;
+    font-weight: 400;
+    margin-right: 5px;
+}
```
Comment (bp323) [inline apps/timetable/admin/ui/css/admin.css]: This doesn't work for multi-line descriptions. What about the following:

```css
#gh-batch-edit-container > i.fa-link {
    font-size: 18px;
    font-weight: 400;
    margin: 11px 10px 0 0;
}
```

and in the HTML add the `pull-left` class to the icon.

File: apps/timetable/admin/ui/css/admin.css [hunk 2]
```diff
@@ -394,6 +394,12 @@
     display: block;
 }
 
+#gh-batch-edit-container > i.fa-link{
```
Comment (bp323) [inline apps/timetable/admin/ui/css/admin.css]: Missing space

**Comment by Coenego** [comment]
Follow-up done. Reassigning to @bp323

**Comment by bp323** [comment]
One follow-up item remaining.

**Comment by Coenego** [comment]
Back to @bp323

**Comment by bp323** [comment]
Code is good to go, I'll sanity check the functionality again once the backend PR is merged.

**Comment by bp323** [comment]
Verified and merged.

**Pull request merged**